docs/install-and-run.md

```diff
@@ -263,40 +275,39 @@ your test and in your application code and use the built-in tools seamlessly.
+contains functions that implement C preprocessor macros that aren't available in Swift. There are
+two ways to do so:
 
-1. Include the `EarlGrey.swift` file in your Swift Test Project. The file contains functions that
-   implement C preprocessor macros that aren't available in Swift.
+#### Using the [earlgrey gem](https://rubygems.org/gems/earlgrey): 
+* We recommend uisng the EarlGrey gem to use EarlGrey with Swift.
 
-  * **If you're using the [earlgrey gem](https://rubygems.org/gems/earlgrey)**
+The EarlGrey gem also copies the `EarlGrey.swift` file to your project directory if you're
+passing the `{swift: true}` parameter to the args for the `configure_for_earlgrey()` method as
+in the below snippet. This is compiled into the tests along with EarlGrey.
 
-      The EarlGrey gem also copies the `EarlGrey.swift` file to your project directory if you're passing the
-      `{swift: true}` parameter to the args for the `configure_for_earlgrey()` method as in the below snippet.
-      You need to manually link this file to your project.
-
-      ```ruby
-      configure_for_earlgrey(installer, PROJECT_NAME, test_target, test_target + '.xcscheme', {swift: true})
-      ```
+```ruby
+configure_for_earlgrey(installer, PROJECT_NAME, test_target, test_target + '.xcscheme', {swift: true})
+```
 
-      EarlGrey currently supports Swift 3.0 and 2.3. While 3.0 is the default, you can also use 2.3 by adding
-      `{swift: true, swift_version: 2.3}`.
+EarlGrey currently supports Swift 3.0 and 2.3. While 3.0 is the default, you can also use 2.3
+by adding `{swift: true, swift_version: 2.3}`.
 
-  * **Manual installation without the [earlgrey gem](https://rubygems.org/gems/earlgrey)**
+* Manual installation:
 
-      In this case you need to download the correct
-      [`EarlGrey.swift`](https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/) file
-      for your Swift version and add it to your test bundle.
+In this case you need to download the correct
+[`EarlGrey.swift`](https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/)
+file for your Swift version and include it in to your test bundle.
 
 Here is an example of a very basic EarlGrey test written in Swift:
 
 ```swift
-import EarlGrey
+import EarlGrey // This adds EarlGrey.swift and the EarlGrey framework.
 
- // Select the button with Accessibility ID "clickMe" and assert it's visible.
- EarlGrey.select(elementWithMatcher: grey_accessibilityID("ClickMe"))
-   .assert(grey_sufficientlyVisible())
+// Select the button with Accessibility ID "clickMe" and assert it's visible.
+EarlGrey.select(elementWithMatcher: grey_accessibilityID("ClickMe"))
+.assert(grey_sufficientlyVisible())
 ```
 
-> For compatibility with Swift, we use `grey_allOfMatchers()` and `grey_anyOfMatchers()` instead of `grey_allOf()`
-and `grey_anyOf()` respectively.
+> For compatibility with Swift 2.3, we use `grey_allOfMatchers()` and `grey_anyOfMatchers()`
```

Comment:
is this no longer an issue on Swift 3? otherwise I don't think it makes sense to say 2.3.